opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -0,0 +1,182 @@
+                     and user-auth                                                                                     | optional | `debug="true" use-authentication="false" use-jmta="true" attempt-interval="3000"`
+| `readmail-test` | Defines the test for reading mail, contains readmail-host, readmail-protocol, and user-auth.       | optional | `debug="true" mail-folder="INBOX" attempt-interval="1000" delete-all-mail="false"`
+|===
+
+.Options for sendmail-test
+[options="header, autowidth"]
+|===
+| Attribute            | Description                                                                                    | Required | Default value
+| `debug`              | Show additional debug output                                                                   | optional | `true`
+| `use-authentication` | Whether to use authentication, in the event it is required                                     | optional | `false`
+| `use-jmta`           | Use the JavaMail Mail Transport Agent                                                          | optional | `true`
+| `attempt-interval`   | Interval in ms between send attempts                                                           | optional | `3000`
+|===
+
+.Options for sendmail-host
+[options="header, autowidth"]
+|===
+| Attribute       | Description                                                                                         | Required | Default value
+| `host`          | The SMTP server address for sending mail                                                            | optional | `${ipaddr}`
+| `port`          | The SMTP server port                                                                                | optional | `25`
+|===
+
+.Options for sendmail-protocol
+[options="header, autowidth"]
+|===
+| Attribute              | Description                                                                                  | Required | Default value
+| `char-set`             | Set the character set                                                                        | optional | `us-ascii`
+| `mailer`               | use smtpsend or some alternate mailer                                                        | optional | `smtpsend`
+| `message-content-type` | Set the message content-type                                                                 | optional | `text/plain`
+| `message-encoding`     | Set the message encoding                                                                     | optional | `7-bit`
+| `quit-wait`            | If set to false, the QUIT command is sent and the connection is immediately closed. If set 
+                            to true (the default), causes the transport to wait for the response to the QUIT command.   | optional | `true`
+| `transport`            | smtp  or smtps                                                                               | optional | `smtp`
```

Comment:
```suggestion
| `transport`            | SMTP  or SMTPS                                                                               | optional | `smtp`
```

Reply:
These are literals.  Should that be `smtp` or `smtps` in that case?

Reply:
lower case if they are literals.